cli/src/test/java/org/jboss/pnc/bacon/cli/AppTest.java

```diff
@@ -1,47 +1,56 @@
 package org.jboss.pnc.bacon.cli;
 
 import org.apache.commons.io.FilenameUtils;
-import org.apache.commons.lang.StringUtils;
-import org.jboss.pnc.bacon.common.exception.FatalException;
 import org.junit.jupiter.api.Test;
 
 import java.io.File;
 
+import static com.github.stefanbirkner.systemlambda.SystemLambda.restoreSystemProperties;
 import static com.github.stefanbirkner.systemlambda.SystemLambda.tapSystemErr;
 import static com.github.stefanbirkner.systemlambda.SystemLambda.tapSystemOut;
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertFalse;
-import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.assertj.core.api.Assertions.assertThat;
 
 class AppTest {
 
     private static final String PNC_TEST_CLASSES = FilenameUtils.separatorsToSystem("pnc/target/test-classes");
 
     @Test
     void testHelp() throws Exception {
-        App app = new App();
-        app.run(new String[] { "-h" });
-        String text = tapSystemOut(() -> assertEquals(0, app.run(new String[] { "-h" })));
-        assertTrue(text.contains("Usage: bacon [-hovV] [-p=<configurationFileLocation>] [--profile=<profile>]"));
+        restoreSystemProperties(() -> {
+            System.setProperty("picocli.ansi", "false");
```

Comment:
Setting `picocli.ansi=false`, otherwise it is possible that ANSI codes apepar int he output and the contains fails. This is also recommended to do by the picocli author https://github.com/remkop/picocli/pull/1166#issuecomment-690001747.

Reply:
I thought you were on Linux now? I think this might be helpful to discuss offline

Reply:
Because all this passed on the ci

Reply:
I have asked the author for clarification, but his own tests is setting this via a `@Rule` which is the JUnit 4 equivalent of this lambda.

Reply:
Author responded with the various cases https://github.com/remkop/picocli/pull/1166#issuecomment-690822211

Reply:
I think you have just set this on two of the tests - would it not be better as a BeforeAll/AfterAll ?

Reply:
Yes, this makes sense, in the sense it should work similar to how `@Rule` works, but it doesn't look like I can put a lambda in those methods. Doesn't the code you want to run have to be inside the lambda since there's no on/off, it just resets it after the lambda completes?